src/WebJobs.Script/Config/WorkerConcurrencyOptions.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class WorkerConcurrencyOptions : Hosting.IOptionsFormatter
+    {
+        public WorkerConcurrencyOptions()
+        {
+            // Setting deafault values
+            LatencyThreshold = TimeSpan.FromMilliseconds(100);
+            AdjustmentPeriod = TimeSpan.FromSeconds(10);
+            CheckInterval = TimeSpan.FromSeconds(1);
+            HistorySize = 10;
+            NewWorkerThreshold = 0.2F;
+            MaxWorkerCount = 10;
```

Comment:
Just so I understand - the max cap on dynamic workers is 10 when WDC is enabled?

Reply:
yes, we are starting with 10 as FUNCTIONS_WORKER_PROCESS_COUNT has 10 max(  https://docs.microsoft.com/en-us/azure/azure-functions/functions-app-settings#functions_worker_process_count)
 it can be changed in future.